src/metabase/query_analysis/native_query_analyzer.clj

```diff
@@ -28,6 +28,23 @@
 (p/import-vars
  [nqa.replacement replace-names])
 
+;; For now, we only support a small set of drivers, so we'll define them all here.
+(doseq [driver [:h2
+                :mysql
+                :postgres
+                :redshift
+                :sqlite
+                :sqlserver]]
+  #_{:clj-kondo/ignore [:discouraged-var]}
```

Comment:
Obviously a smell to do this here, but the value of keeping the definition of all the drivers in one place for now trumps it in my opinion.